ParabolInc/parabol
# feat(sprint-poker): Track GitLab events

Fixes https://github.com/ParabolInc/parabol/issues/6303

To test: 

- Copy the `SEGMENT_WRITE_KEY` and `SEGMENT_FN_KEY` from the production env vars in the devops repo. Paste them in your `.env` file
- Go to Amplitude and search for the user with your `viewerId`. Select on the user and see the GitLab events 
- Comment out the Segment env vars once you've finished testing

This PR adds event tracking in GitLab to the following:

- [x] The start of a search
- [x]  When a user selects an issue in the Scope phase
- [x]  When a project filter menu item is selected
- [x]  When the select all button is used
- [x]  When a search input has been cleared

This PR tracks when a project has been created: https://github.com/ParabolInc/parabol/pull/6332/
